### ISSUE TITLE
[ResourceBundle] subtree split out of date.

### ISSUE THREAD

**Issue by bendavies** [Sylius/Sylius#3636]
Looks like the subtree split for the [ResourceBundle](https://github.com/Sylius/SyliusResourceBundle) hasn't completed in a while?

The latest commit it showing as 29 days ago.

**Comment by pjedrzejewski** [comment]
Looking into it.

**Comment by tristanbes** [comment]
same for FlowBundle changes merged in https://github.com/Sylius/Sylius/pull/3635 are not present in the subtree

**Comment by pjedrzejewski** [comment]
@tristanbes Subtree split is running fine for all bundles except ResourceBundle it seems, that's why I missed it. :/ It runs every 12 hours. (I want to make it run on every change, but that's some work)

I am investigating the issue with SyliusResourceBundle.

**Comment by bendavies** [comment]
thanks for the update @pjedrzejewski

**Comment by liverbool** [comment]
:sob:

**Comment by gperdomor** [comment]
any progress here? :(

**Comment by pjedrzejewski** [comment]
Fixed! Sorry but I had to figure it out without force pushing ResourceBundle. :) For some reason it got 2 commits which were not in the Sylius/Sylius repo, thus subtree split was broken.

I could not identify what it happened though. That being said, I need to do some work on the whole subtree split, it's performance and the way it is run. Right not it is every 12 hours, but I would like it to run on every merge only for specific components/bundles.
